tools/ManifestSignerClient/ManifestSignerClient/CertificateUtil.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Cryptography;
+using Org.Webpki.JsonCanonicalizer;
+using Newtonsoft.Json.Linq;
+
+namespace ManifestSignerClient
+{
+    public class CertificateUtil
+    {
+        static public string GetBase64CertContent(string certPath)
```

Comment:
This function and the other two (GetDerECDsaPrivateKeyFromPem, GetRSAPrivateKeyFromPem) have quite similar code checks and patterns. Any chance we can create a utility function to reuse?